frontend/src/app/modules/user-card/mappingLanguagetoCountry.ts

```diff
@@ -0,0 +1,188 @@
+interface CountryCodes {
+    [key: string]: string;
+}
+
+export const countryCodes: CountryCodes = {
+    aa: 'ET', // Afar
+    ab: 'GE', // Abkhazian
+    ae: 'AE', // Avestan
+    af: 'ZA', // Afrikaans
+    ak: 'GH', // Akan
+    am: 'ET', // Amharic
+    an: 'ES', // Aragonese
+    ar: 'SA', // Arabic
+    as: 'IN', // Assamese
+    av: 'RU', // Avaric
+    ay: 'BO', // Aymara
```

Comment:
we should have this in the backend

Reply:
and why do we need codes? I believe we already have countries in the backend

Reply:
In this task, i need to display country flag, only by language ,i didn't found any libraries where all languages are mapped to countries so i created a map for this, and it's only for visual, i don't know if we need this in backend